TECHNOLOGY_CHOICES.md

```diff
@@ -2,10 +2,17 @@
 # Database
 ## Checks for determining database
 - Database should be able to store XML data
-- Database should be relational, because we need relations between files/pieces of files/models
+- Database should be relational, because we need relations between files/pieces of files/models (not sure about this one)
```

Comment:
relations between files/pieces of files/models are required this the requirement; How this could be realized depends on the database technology

Reply:
With the use of xQuery, a lot of the relational requirements can be fulfilled.
For a small introduction, take a look at this YouTube video about how xQuery compares to SQL:
https://www.youtube.com/watch?v=NUTvFwXGN-M